ssl/statem/statem_srvr.c

```diff
@@ -1191,13 +1178,13 @@ MSG_PROCESS_RETURN tls_process_client_hello(SSL *s, PACKET *pkt)
     s->statem.cleanuphand = 1;
 
     /*
-     * First, parse the raw ClientHello data into the CLIENTHELLO_MSG structure.
+     * First, save a pointer to the raw ClientHello data for the early callback,
+     * and parse it into the CLIENTHELLO_MSG structure.
```

Comment:
Also need to revert this comment back, if we keep the commits that remove access to the raw ClientHello octets.